docs/sphinx/dev_guide/contributing.rst

```diff
@@ -193,6 +193,63 @@ likely the develop branch.
                **username/feature/<name-of-feature>** for a new feature, 
                **username/bugfix/<issue-fixed>** for a bugfix, etc.
 
+-----------------------------------
+Style
+-----------------------------------
+RAJA enforces style within the ``src`` and ``include`` directories using clang-format,
+major version 14.  Formatting will be enforced on pull requests through CI.
+The contributor must guarantee that their code is in compliance with the
+clang-format settings specified within ``.clang-format``.  To make this easy,
+RAJA has provided two options for applying clang-format:
+
+* The CMake build target ``style``
+
+  *  If a ``clang-format`` executable with major version 14 is available in the ``PATH``
+     when running CMake, RAJA's build system should find that executable and set 
+     the CMake variable ``CLANGFORMAT_EXECUTABLE`` to that executable's path.  
+  *  If an invalid version exists in the ``PATH``, or no ``clang-format`` version is 
```

Comment:
I think this second bullet can be more simply and clearly stated by simply saying something like this....

Alternatively, the `CLANGFORMAT_EXECUTABLE` CMake variable can be set by the user to the path of the desired clang-format.  For example, ...

As it is, it combines several things that may not be easy for a reader to sort out.

Reply:
good point